README.rst

```diff
@@ -22,6 +22,8 @@ Requirements
 ------------
 
 For IPMB interface a `Total Phase`_ Aardvark is needed.
+Another option is to use ipmb-dev-int driver on Linux with an I2C bus, driver of which supports slave mode:
+https://www.kernel.org/doc/html/latest/driver-api/ipmb.html
 
```

Comment:
I think this supports a bit more documentation. I just had a quick glance at /dev/ipmb-N, and it seems to be a slave interface, but you just use it to do a request, right? This should be documented.

Reply:
Yes, it does support sending and receiving both requests and responses:
https://lists.ozlabs.org/pipermail/linux-aspeed/2019-November/002879.html